test-vectors/did_web/resolve.json

```diff
@@ -70,6 +75,8 @@
         "didUri": "did:web:doesnotexist.com"
```

Comment:
What are your thoughts on returning `internalError` instead of `notFound` when the HTTP GET fails?

Seems like that's the direction based on reading this: https://github.com/TBD54566975/web5-kt/pull/172/files#diff-51da16f10d7795c05d587a710d194142c4addebe8657e5abd9570ac6fc4cc738R64-R73

Reply:
@frankhinek did you mean to link to a commit done within this PR?

My thoughts are that notFound carries a different semantic meaning than internal error. Programmatically, I'd expect retries to be done with internal errors. I wouldn't expect that to happen with not found. 

Given that, I'd like to keep as is. I'm open to hearing other opinions. Can you share your rationale?